src/dynarec/rv64/dynarec_rv64_00_0.c

```diff
@@ -584,10 +584,10 @@ uintptr_t dynarec64_00_0(dynarec_rv64_t* dyn, uintptr_t addr, uintptr_t ip, int
             SETFLAGS(X_ALL, SF_SET_PENDING, NAT_FLAGS_FUSION);
             i64 = F32S;
             if (i64) {
-                MOV64xw(x2, i64);
+                MOV64xws(x2, i64);
```

Comment:
why not simply using `MOV64x(x2, i64)` then?

Reply:
sorry 😅